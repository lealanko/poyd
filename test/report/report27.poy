(* Build and report trees : total *)
read ("1.fas", "30.ss" )
build (1)
report (trees: (newick))
