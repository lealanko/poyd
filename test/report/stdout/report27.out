(t1,(t2,((t29,((t28,(t27,(t26,t25))),((t46,(t6,t49)),(t33,(t40,(t11,(t47,t38)
)))))),((t16,(t15,(t45,t35))),((((t48,t13),(t37,(t14,t12))),((t32,(t9,(t8
,t36))),(t3,(t4,(t5,(t7,t34)))))),((t22,((t44,t43),(t41,(t19,(t30,(t20,t18)))
))),((t31,t23),((t24,t21),((t42,t17),(t39,t10))))))))));